### DISCUSSION TITLE
use docker build in npm or yarn commend fail (exec /bin/sh: exec format error)

### DISCUSSION THREAD

**Discussion by imagine10255** [lima-vm/lima#956] [Q&A]
use docker build in npm or yarn commend fail

<img width="979" alt="CleanShot 2022-07-13 at 16 19 25@2x" src="https://user-images.githubusercontent.com/5771353/178685861-875fef53-6dea-48d6-903d-8cf0f61de344.png">

use docker desktop for mac is success, but lima is fail~
What do I need to pay attention to?

os: macOs Monterey 12.4

**Comment by afbjorklund** [comment]
Make sure that you get the right architecture, that looks like an amd64 image (trying to run on an arm64 host ?)

You can install `qemu-user-static` and run in emulation, but it is better to get the native architecture to start with.

**Reply by imagine10255** [reply]
not work..
